create the skeleton for the public notebook. (#122)
hassanobeid1994/tr_b_causal_2020: notebooks/final/7.0-mab-selection-on-observables-final.md

---
jupyter:
  jupytext:
    formats: ipynb,md
    text_representation:
      extension: .md
      format_name: markdown
      format_version: '1.2'
      jupytext_version: 1.5.1
  kernelspec:
    display_name: Python 3
    language: python
    name: python3
---

# Selection on Observables 

## Purpose 

> The purpose of this notebook is to illustrate an example of the workflow outlined in [Brathwaite and Walker (2017)](https://arxiv.org/abs/1706.07502). This simple application aims at highlighting the importance of causal structure in estimating causal effects of interest reflecting changes resulting from policy proposals. The basic idea is to show that when we control for intermediate variables of some variable of interest in a causal graph, we never recover the true causal parameter on the variable of interest.

Reviewer: nit: Can you put each sentence on a separate line? It makes git diffs more meaningful as it'll allow highlighting just the part of the sentence that changed instead of the whole line containing unchanged sentences.

I think that the purpose of the notebook is to show if one ignores the structure of one's causal graph (i.e., one's treatment assignment mechanism), one can compute heavily biased treatment effect estimates. We contrast treatment effect estimation while using a correct causal graph vs using the naive, all-variables-are-independent default causal graph.

> This notebook achieves this objective by using the dataset and the MNL utility specification from [Brathwaite and Walker (2016)](https://arxiv.org/abs/1606.05900).

Reviewer: ```suggestion
This notebook uses the dataset and the MNL utility specification from [Brathwaite and Walker (2016)](https://arxiv.org/abs/1606.05900) for demonstration.
```

The rest of the notebook is organized as follows:

>  - Defining different causal graphs representing different views for how individuals make mode choices. These causal graphs are based on the MNL utility function from [Brathwaite and Walker (2016)](https://arxiv.org/abs/1606.05900)

Reviewer: ```suggestion
 - Defining different causal graphs representing different views for how individuals make mode choices. These causal graphs are based on the MNL utility functions from [Brathwaite and Walker (2016)](https://arxiv.org/abs/1606.05900)
```

 - Simulating data based on the different beliefs about the data generating process illustrated by both causal graphs.

>  - Perturbing one of the variables (e.g.: Travel Distance) to imitate scenarios where a policy proposal would affect one or more of the variables represented by nodes in the causal graph.

Reviewer: ```suggestion
 - Perturbing one of the variables (e.g.: Travel Distance) to simulate a policy intervention.
```

>  - Estimating choice models resulting from the different causal graphs and plotting the distribution of average causal effects of the variable perturbation from the previous step

Reviewer: ```suggestion
 - Calculating and plotting the distributions of treatment effects according to different causal graphs. 
```
